components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -280,6 +280,8 @@ function cleanup_kfp() {
     # If this fails, deleting the nodegroup later will clean it up anyway
     kill -9 $MINIO_PID || true
   fi
+  kubectl delete -k "github.com/kubeflow/pipelines/manifests/kustomize/env/cert-manager/dev?ref=$KFP_VERSION&timeout=90s"
```

Comment:
How long does this line take? Do we need to add wait time?

Reply:
About 2 minutes at the maximum since this is a lightweight distribution, wait time is unnecessary here. My concern was more about the installation, it may take longer because of needing to create the persistent volume, which depends on EBS. I think the best solution might just be to uninstall on test failure.

Reply:
Addressed in new revision, uninstallation only occurs on failure which mitigates the risk of a long uninstallation/installation.